app/App.js

```diff
@@ -4,6 +4,7 @@ import {
   Text,
   View,
   StyleSheet,
+  // CR * understand why unused
```

Comment:
My linter reported PixelRatio and other imports here as unused. I think it's important, eventually, to understand why they are not unused and also remove any unused imports. Cleaner code! ❇️